heron/shell/src/python/BUILD

```diff
@@ -7,7 +7,7 @@ pex_library(
     ),
     reqs = [
         "requests==2.12.3",
-        "tornado==4.0.2",
+        "tornado==4.5.3",
```

Comment:
tornado is still needed?

Reply:
both `heron-shell` and `heron-tracker` use tornado still, this PR just removes the use of it in explorer+UI. I forgot to remove `tornado` from the reqs of the explorer after rebasing though